src/v2/mod.rs

```diff
@@ -1057,6 +1151,22 @@ impl IntoBlockIdentifier for RelativeBlockHeight {
     fn into_block_identifier(self) -> BlockIdentifier { BlockIdentifier::RelativeHeight(self) }
 }
 
+pub trait IntoEpochIdentifier {
```

Comment:
Documentation.

I'm not so convinced that this trait makes sense. Why do we need it?

Reply:
Removed it.

Reply:
Some zombie version of it appears to have surived :) 

The trait is still there.